pkg/tenant/db/tenants.go

```diff
@@ -12,6 +13,9 @@ import (
 
 const TenantNamespace = "tenants"
 
+// Tenant names must be URL safe and begin with a letter.
+var TenantNameRegex = regexp.MustCompile("^[a-zA-Z][a-zA-Z0-9.-]*$")
```

Comment:
https://regex101.com/r/FR4U1I/1

```suggestion
var TenantNameRegex = regexp.MustCompile(`^[a-zA-Z_][a-zA-Z0-9\.\-_]*$`)
```